From randy.dunlap@oracle.com  Fri Jun  6 14:29:31 2008
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Wed, 4 Jun 2008 21:40:43 -0700
Subject: dev_set_name: fix missing kernel-doc
Cc: gregkh <greg@kroah.com>, akpm <akpm@linux-foundation.org>
Message-ID: <20080604214043.a29464a0.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

Fix kernel-doc for new dev_set_name() function:

Warning(lin2626-rc5//drivers/base/core.c:767): No description found for parameter 'fmt'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/core.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/base/core.c
+++ b/drivers/base/core.c
@@ -762,6 +762,7 @@ static void device_remove_class_symlinks
 /**
  * dev_set_name - set a device name
  * @dev: device
+ * @fmt: format string for the device's name
  */
 int dev_set_name(struct device *dev, const char *fmt, ...)
 {
